packages/wonder-blocks-layout/components/media-layout.test.js

```diff
@@ -0,0 +1,343 @@
+                return <View style={styles.test}>Hello, world!</View>;
+            }}
+        </MediaLayout>,
+    );
+    const style = wrapper.find("div").prop("style");
+
+    return style;
+};
+
+global.SNAPSHOT_INLINE_APHRODITE = true;
+
+describe("MediaLayout", () => {
+    beforeEach(() => {
+        unmountAll();
+        window.matchMedia = matchMedia;
+    });
+
+    describe("mediaSize", () => {
+        it("should be small when viewport width < 768", async () => {
+            // Arrange
+            resizeWindow("small");
+
+            // Act
+            const args = await getArgsToChildren();
+
+            // Assert
+            expect(args.mediaSize).toEqual("small");
+        });
+
+        it("should be medium when viewport 768 < width < 1024", async () => {
+            // Arrange
+            resizeWindow("medium");
+
+            // Act
+            const args = await getArgsToChildren();
+
+            // Assert
+            expect(args.mediaSize).toEqual("medium");
+        });
+
+        it("should be medium when viewport width > 1024", async () => {
+            // Arrange
+            resizeWindow("large");
+
+            // Act
+            const args = await getArgsToChildren();
+
+            // Assert
+            expect(args.mediaSize).toEqual("large");
+        });
+    });
+
+    describe("styleSheets", () => {
+        const testSizes = {
+            small: [640, 480],
+            medium: [800, 600],
+            large: [1200, 800],
+        };
+
+        for (const size of Object.keys(testSizes)) {
+            it(`should always provide styles from all (${size})`, async () => {
+                // Arrange
+                const styleSheets = {
+                    all: StyleSheet.create({
+                        test: {
+                            color: "blue",
+                        },
+                    }),
+                };
+                resizeWindow(size);
+
+                // Act
+                const style = getStyle(styleSheets);
+
+                // Assert
+                expect(style.color).toBe("blue");
+            });
+
+            it(`"mdOrSmaller" should match ${
+                size === "large" ? "not" : ""
+            } "${size}"`, async () => {
+                // Arrange
+                const styleSheets = {
+                    mdOrSmaller: StyleSheet.create({
+                        test: {
+                            color: "blue",
+                        },
+                    }),
+                    large: StyleSheet.create({
+                        test: {
+                            color: "orange",
+                        },
+                    }),
+                };
+                resizeWindow(size);
+
+                // Act
+                const style = getStyle(styleSheets);
+                const color = {
+                    small: "blue",
+                    medium: "blue",
+                    large: "orange",
+                }[size];
```

Comment:
This `color` variable should be in the arrange section, right? And probably named `expectation`?